framework/doc/content/source/dgkernels/ArrayHFEMDiffusion.md

```diff
@@ -0,0 +1,14 @@
+# ArrayHFEMDiffusion
+
+!syntax description /DGKernels/ArrayHFEMDiffusion
+
+## Overview
+
+This Array kernel applies the constraints on internal sides for diffusion equations with a hybrid finite element method using an array variable.
+Its standard variable version is [HFEMDiffusion].
```

Comment:
Are we missing `.md` here? We will see when civet posts the link for the updated webpages of this PR.

Reply:
I think it should be " Its standard variable version is [HFEMDiffusion.md]"